### DISCUSSION TITLE
development updates?

### DISCUSSION THREAD

**Discussion by fast2611** [rvmn/SuperPleccer#18] [Q&A]
Hi! 
Been following progress for a few months now and been wondering if new versions (or rather windows builds since I know nothing of compiling code myself <.<) are still in the works, been a huge fan of the groundbreaking overhang features!
(Hope this is the right place to ask!)

**Comment by rvmn** [comment]
yes, i am now busy trying to make the windows build work and next week will try to finish up on the bridging upgrade to make bridging optimized in sense of direction and auto-support pillar placement. expect things to come in the next week, at least an announcement. 
and yes, this is absolu the right place to ask!

**Comment by rvmn** [comment]
Please fill in the [poll](https://github.com/rvmn/SuperPleccer/discussions/27)
Thank you!
